CryptKeeperZK/crypt-keeper-extension
# feat: mnemonic data encryption

## Explanation

This PR changes encryption/decryption logic for data and keeps mnemonic inside storage.
Note: this is a breaking change, need to reinstall the extension.

Details are below:
- [x] Encrypt/decrypt data with mnemonic
- [x] Encrypt/decrypt mnemonic with user password

## More Information

Closes #572 
Closes #585 

## Screenshots/Screencaps

N/A

## Manual Testing Steps

1. Reinstall the extension
2. Check onboarding, login flows
3. Check identity creation, history.

## Pre-Merge Checklist


> PR template source from [github.com/MetaMask](https://github.com/MetaMask)
